lib/datadog_backup/core.rb

```diff
@@ -38,7 +39,7 @@ def diff(id)
     def except(hash)
       hash.tap do # tap returns self
         @banlist.each do |key|
-          hash.delete(key) # delete returns the value at the deleted key, hence the tap wrapper
+          hash ? hash.delete(key) : {} # delete returns the value at the deleted key, hence the tap wrapper
```

Comment:
these values do not return, and are side effects used to mutate `hash`. I'm not sure what the ternary operator would do here?